.github/workflows/readme-update.yml

```
@@ -0,0 +1,271 @@
+                if (firstLine && !firstLine.startsWith('--')) {
+                  const diagType = firstLine.toLowerCase();
+                  let title = 'Diagram';
+                  if (diagType.includes('graph')) title = 'Graph Diagram';
+                  if (diagType.includes('flowchart')) title = 'Flowchart';
+                  if (diagType.includes('sequence')) title = 'Sequence Diagram';
+                  if (diagType.includes('gantt')) title = 'Gantt Chart';
+
+                  modified = true;
+                  return \`\`\`mermaid
+accTitle: \${title}
+\${diagram.trim()}
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)